content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -74,10 +213,39 @@ Contact the Restack team via chat to set a custom domain for your Kestra instanc
 
 Kestra will be deployed on your cluster and you can access it using the link under the *URL* tab.
 ![lightdash-deployed](/docs/administrator-guide/deployment/restack/kestra-deployed.png)
-You can also check the workloads and volumes that are deployed within Kestra.
-![access-lightdash](/docs/administrator-guide/deployment/restack/kestra-access.png)
 
-## Deploy multiple instances of Kestra
+## Deploy your Kestra image
+
+Activate **Restack CI/CD previews** under Stack > Settings to deploy your own Airflow image by connecting your GitHub repository.
+
+With CI/CD preview activated, every time you make a pull request to your GitHub repo, Restack takes every file and deploy them into your cloud. 
+
+For every pull request, Restack provides you a URL with a preview environment for you to review. Once that you merge your pull request into main, Restack flushes the preview environments, and builds the repo into your production environment.
+
+Here you can find the ![Kestra starter repo](https://github.com/restackio/starter-kestra).
```

Comment:
```suggestion
For every pull request, Restack provides you with a URL with a preview environment, allowing you to review the pull request. Once you merge your pull request, Restack flushes the preview environments and builds the deployed code from the repository into your production environment.

The following [Kestra Restack repository](https://github.com/restackio/starter-kestra) can help you get started.
```

cut `main` as I assume people can customize the branch name, WDYT? or is it always main?